docs/img/pandas_on_ray_data_ingress.svg

```diff
@@ -0,0 +1,4 @@
393" y="18" fill="rgba(0, 0, 0, 1)" font-family="Helvetica" font-size="12px" text-anchor="middle">Query for IO class&#xa;</text></switch></g><path d="M 580 33 L 631.88 33" fill="none" stroke="rgba(0, 0, 0, 1)" stroke-miterlimit="10" pointer-events="stroke"/><path d="M 641.88 33 L 631.88 36.33 L 631.88 29.67 Z" fill="rgba(0, 0, 0, 1)" stroke="rgba(0, 0, 0, 1)" stroke-miterlimit="10" pointer-events="all"/><path d="M 626 97 L 626 77.95 L 549.42 77.95 L 549.93 61.23" fill="none" stroke="rgba(0, 0, 0, 1)" stroke-miterlimit="10" stroke-dasharray="3 3" pointer-events="stroke"/><path d="M 554.66 70.27 L 549.97 60.12 L 544.66 69.96" fill="none" stroke="rgba(0, 0, 0, 1)" stroke-miterlimit="10" pointer-events="all"/><path d="M 118 33 L 199.88 33" fill="none" stroke="rgba(0, 0, 0, 1)" stroke-miterlimit="10" pointer-events="stroke"/><path d="M 209.88 33 L 199.88 36.33 L 199.88 29.67 Z" fill="rgba(0, 0, 0, 1)" stroke="rgba(0, 0, 0, 1)" stroke-miterlimit="10" pointer-events="all"/><path d="M 329 33 L 451.88 33" fill="none" stroke="rgba(0, 0, 0, 1)" stroke-miterlimit="10" pointer-events="stroke"/><path d="M 461.88 33 L 451.88 36.33 L 451.88 29.67 Z" fill="rgba(0, 0, 0, 1)" stroke="rgba(0, 0, 0, 1)" stroke-miterlimit="10" pointer-events="all"/><path d="M 417 97 L 417 77.95 L 495.46 77.95 L 495.94 61.24" fill="none" stroke="rgba(0, 0, 0, 1)" stroke-miterlimit="10" stroke-dasharray="3 3" pointer-events="stroke"/><path d="M 500.68 70.26 L 495.97 60.12 L 490.69 69.97" fill="none" stroke="rgba(0, 0, 0, 1)" stroke-miterlimit="10" pointer-events="all"/><path d="M 522 97 L 522 84.54 L 522 61.24" fill="none" stroke="rgba(0, 0, 0, 1)" stroke-miterlimit="10" stroke-dasharray="3 3" pointer-events="stroke"/><path d="M 527 70.12 L 522 60.12 L 517 70.12" fill="none" stroke="rgba(0, 0, 0, 1)" stroke-miterlimit="10" pointer-events="all"/><path d="M 194.01 117 L 178 117 L 178 142.8 L 147.84 142.8" fill="none" stroke="rgba(0, 0, 0, 1)" stroke-miterlimit="10" pointer-events="stroke"/><path d="M 209.01 117 L 201.51 121.41 L 194.01 117 L 201.51 112.59 Z" fill="none" stroke="rgba(0, 0, 0, 1)" stroke-miterlimit="10" pointer-events="all"/><path d="M 159.72 136.3 L 146.72 142.8 L 159.72 149.3" fill="none" stroke="rgba(0, 0, 0, 1)" stroke-miterlimit="10" pointer-events="all"/><path d="M 271 72.99 L 271 94.76" fill="none" stroke="rgba(0, 0, 0, 1)" stroke-miterlimit="10" pointer-events="stroke"/><path d="M 271 57.99 L 275.41 65.49 L 271 72.99 L 266.59 65.49 Z" fill="none" stroke="rgba(0, 0, 0, 1)" stroke-miterlimit="10" pointer-events="all"/><path d="M 264.5 82.88 L 271 95.88 L 277.5 82.88" fill="none" stroke="rgba(0, 0, 0, 1)" stroke-miterlimit="10" pointer-events="all"/></g><switch><g requiredFeatures="http://www.w3.org/TR/SVG11/feature#Extensibility"/><a transform="translate(0,-5)" xlink:href="https://www.diagrams.net/doc/faq/svg-export-text-problems" target="_blank"><text text-anchor="middle" font-size="10px" x="50%" y="100%">Viewer does not support full SVG 1.1</text></a></switch></svg>
```

Comment:
Is it intended that this line goes straightforward from the Dispatcher to the IO-class and not from the actual factory to the IO-class?
![image](https://user-images.githubusercontent.com/62142979/144018645-a02c9a7d-5fa9-4b93-a715-e4c91dd552a7.png)

Reply:
Yes, it is intended and we say the following
```
When a user calls any IO function from the modin.pandas.io module, the API layer queries the FactoryDispatcher which defines a factory specific for the execution, namely, the PandasOnRayFactory. The factory, in turn, exposes the PandasOnRayIO class whose responsibility is to perform a parallel read/write from/to a file.
```